### PR TITLE
Render summarized ort perf with tree map in browser

### PR DESCRIPTION
**Pull request by RandySheriffH**
Render ort perf number in a json file into browser.
First, open the json file:
image
One will see:
image
And each op is clickable to see detailed calling instances:
image

### CODE REVIEW

**Comment by fs-eire** [comment]
It's a good idea to visualize the perf result. 2 concerns:
1. have a readme or markdown doc for easy instructions
2. allows a commandline launcher (eg. `ort_perf_view ./my_perf.json` which launches a browser window)

File: tools/perf_view/ort_perf_view.html [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+              document.body.style.cursor = "wait";
+              return true;
+          });
+          chart_cpu.listen("chartDraw", function(e){
+              console.log('cpu refreshed');
+              document.body.style.cursor = "default";
+              return true;
+          });
+
+          //chart_cpu.hintDepth(1);
```
Comment (zhangyaobit) [inline tools/perf_view/ort_perf_view.html:115]: Do we need this?

File: tools/perf_view/ort_perf_view.html [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+          //chart_cpu.hintOpacity(0.8); 
+          chart_cpu.container("cpu");
+          chart_cpu.draw();
+          
+           if (has_gpu_info) {
+            document.getElementById("cpu").className = "half_view";
+            var chart_gpu = anychart.treeMap(anychart.data.tree(array_gpu, "as-tree"));
+            chart_gpu.title("GPU perf overview");
+            //chart_gpu.hintDepth(1);
```
Comment (zhangyaobit) [inline tools/perf_view/ort_perf_view.html:124]: Same as above.

File: tools/perf_view/ort_perf_view.html (hunk 1 above)
Comment (RandySheriffH) [reply tools/perf_view/ort_perf_view.html:115]: Yes - we could see a bettered transparent view of deeper layer when enabling it.
The reason for commenting it out for now is that it causes high latency on bigger json files, will optimize that when got a chance.

File: tools/perf_view/ort_perf_view.html (hunk 2 above)
Comment (RandySheriffH) [reply tools/perf_view/ort_perf_view.html:124]: Same as replied above - let's just keep it.

**Review by zhangyaobit — APPROVED** [review]

**Pull request merged**